src/app/components/Hero/package.json

```diff
@@ -1,4 +1,4 @@
 {
-    "main": "Hero.js",
-    "storybook": "HeroStory.js"
+    "name": "Hero.js",
```

Comment:
Since `name` is a required field we should probably have it, but I think you meant to leave off the `.js` like you did in most of the others?